src/server/dragonfly_test.cc

```diff
@@ -898,6 +898,15 @@ TEST_F(DflyEngineTest, Issue679) {
   Run({"EXPIRE", "a", "1001"});
 }
 
+TEST_F(DflyEngineTest, Issue742) {
+  // https://github.com/dragonflydb/dragonfly/issues/607
+  // The stack was not cleaned in case of an error and it blew up.
+  for (int i = 0; i < 3'000; i++) {
+    Run({"EVAL", "redis.get(KEYS[1], KEYS[2], KEYS[3], KEYS[4], KEYS[5])", "5", "k1", "k2", "k3",
+         "k4", "k5"});
+  }
```

Comment:
I experimentally determined this is the min amount to make it fail. We need to pollute the stack largely enough